db/migrate/201805145603_fix_template_audit_versions.rb

```diff
@@ -0,0 +1,15 @@
+class FixTemplateAuditVersions < ActiveRecord::Migration[5.1]
+  def up
+    ['ProvisioningTemplate', 'Ptable'].each do |model|
+      model.constantize.unscoped.each do |object|
```

Comment:
you can use `[ ProvisioningTemplate, Ptable ].each` to avoid constantize call later.